CSC322-Grinnell/mayflower-directory
# Update and rename the default profile photo

# Changes

- Use a better background for the default profile photo
- Rename the default profile photo to be more specific

# Testing

1. Start the server
2. Go to directory page.  The new default photo should match better with existing resident photos
